src/obsidian_ai_hub/coding/backend.py

```
@@ -247,6 +248,64 @@ def execute(
+                        if "text" in data and isinstance(data["text"], str):
+                            text_parts.append(data["text"])
+                        elif "content" in data and isinstance(data["content"], str):
+                            text_parts.append(data["content"])
+                        elif "message" in data:
+                            msg = data["message"]
+                            if isinstance(msg, str):
+                                text_parts.append(msg)
+                            elif isinstance(msg, dict) and "content" in msg:
+                                text_parts.append(str(msg["content"]))
+                except Exception:
+                    pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)